site/layouts/shortcodes/alpha.html

```diff
@@ -0,0 +1,9 @@
+{{- $gate := .Get "gate" }}
+{{- $title := .Get "title" }}
+<div class="alert alert-warning" role="alert">
+    <h4 class="alert-heading">Warning</h4>
+    <p>The {{ $title }} feature is currently <strong><a href="{{ ref . "/docs/Guides/feature-stages.md#alpha" }}">Alpha</a></strong>,
+        not enabled by default, and may change in the future.</p>
+    <p>Use the Feature Gate <code>{{- $gate }}</code> to enable and test this feature.</p>
+    <p>See the <a href="{{ ref . "/docs/Guides/feature-stages.md#alpha" }}">Feature Gate documentation</a> for details on how to enable features.</p>
```

Comment:
The two links here go to the same place. I'm not sure that page actually helps me if I want to enable this feature gate (indicating that we might need some more documentation around that particular scenario).

Also, I noticed that player tracking isn't in the table at the top of that page: https://773f473-dot-preview-dot-agones-images.appspot.com/site/docs/guides/feature-stages/

Reply:
This link might be more useful for the second case:
https://773f473-dot-preview-dot-agones-images.appspot.com/site/docs/guides/feature-stages/#feature-gates
```
Alpha and Beta features can be enabled or disabled through the agones.featureGates configuration option that can be found in the Helm configuration documentation.
```

Reply:
> Also, I noticed that player tracking isn't in the table at the top of that page: https://773f473-dot-preview-dot-agones-images.appspot.com/site/docs/guides/feature-stages/

That's the next PR. I have a Player Tracking actual guide that I wanted to link from there, but also wanted this page first to point to. This one seemed long enough on it's own to review.

> The two links here go to the same place.
Ah! Good catch both of you. I think I meant that to go to #feature-gates and fumbled it. Making that change.